nncf/torch/dynamic_graph/graph_tracer.py

```diff
@@ -108,8 +108,6 @@ def trace_graph(self, model: torch.nn.Module, context_to_use: Optional['TracingC
                     self.custom_forward_fn(model)
         model.load_state_dict(sd)
 
-        if isinstance(model, PostGraphBuildActing):
-            model.post_build_graph_actions()
```

Comment:
`PostGraphBuildActing.post_build_graph_actions` no used anymore, looks like class `PostGraphBuildActing` can be removed

Reply:
Thanks for spotting this, removed